core/src/mindustry/content/Blocks.java

```diff
@@ -1754,7 +1755,7 @@ public void load(){
             size = 4;
             shootCone = 2f;
             shootSound = Sounds.railgun;
-            unitSort = (u, x, y) -> -u.maxHealth;
+            unitSort = (u, x, y) -> -u.maxHealth + Mathf.dst2(u.x, u.y, x, y) / 800f;
```

Comment:
Considering how high dst2 can get, this should be much higher - around 5k, maybe.